OurTimeForTech/shiftwork2
# Custom 404 and 505 pages

## What type of PR is this? (check all applicable)

- [] Feature
- [] Refactor
- [] Bug Fix
- [] Documentation Update
- [x] Other (describe: Styling)


## Description of what PR does

- Removed 404 and 505 html file in Public folder and Generate a new ErrorsController that inherits from ApplicationController with command `rails g controller errors not_found internal_server_error.`
- Configure Routes to Handle New Errors Page in `config/routes.rb.`
- Adjust configuration to turn off the default way Rails will handle errors in the main `application.rb` file.
- Disable configuration in `config/environments/development.rb` in order to test things out locally.
- Style, edit and adjust the views within `app/views/errors` accordingly.


## Related PRs and/or Issues (if any)
- #24 
-


## QA Instructions, Screenshots, Recordings
For example, go to http://127.0.0.1:3000/sss to see how error page looks like.
https://web-crunch.com/posts/custom-error-page-ruby-on-rails
## Added tests?



## Added to documentation?
